src/HW/client/src/components/AppContainer/AppContainerConnector.js

```diff
@@ -0,0 +1,22 @@
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import AppContainer from "./AppContainer";
+import { getItemsAction } from "../../redux/actions/itemsEntitiesActions";
+import { getItems,getTasksStatusState } from "../../redux/selectors/itemsEntitiesSelectors";
+
+const mapStateToProps = (state, ownProps) => {
+    const numOfItems = getItems(state).length
+    const tasksStatusState = getTasksStatusState(state)
+  return {numOfItems,tasksStatusState};
+};
+
+const mapDispatchToProps = (dispatch, ownProps) => {
+  return bindActionCreators(
+    {
+      getItemsAction,
+    },
+    dispatch
+  );
+};
```

Comment:
you can also use this shorten -

```suggestion
const mapDispatchToProps = {
      getItemsAction,
};
```
if the only thins you do is use bindActionCreators , it will be the default behavior when adding the object of all functions needed to bind